sql-plan-management.md

```diff
@@ -479,6 +484,121 @@ SHOW binding_cache status;
 1 row in set (0.00 sec)
 ```
 
+## Cross-database binding
+
+In the SQL statement used to create a binding, TiDB supports cross-database binding using the wildcard `*` to represent a database. This feature is introduced in v7.6.0. To use the cross-database binding feature, you need to first enable the [`tidb_opt_enable_fuzzy_binding`](/system-variables.md#tidb_opt_enable_fuzzy_binding-new-in-v760) system variable.
+
+You can use cross-database binding to simplify the process of fixing execution plans in scenarios where data is categorized and stored across different databases, and each database maintains identical object definitions and executes similar application logic. The following are some common use cases:
+
+* When you run SaaS or PaaS services on TiDB, where the data of each tenant is stored in separate databases for easier data maintenance and management.
+* When you performed database sharding in a single instance and retained the original database schema after migrating to TiDB, that is, the data in the original instance is categorized and stored by database.
+
+In these scenarios, cross-database binding can effectively mitigate SQL performance issues caused by uneven distribution and rapid changes in user data and workload. SaaS providers can use cross-database binding to fix execution plans validated by applications with large data volumes, thereby avoiding potential performance issues due to the rapid growth of applications with small data volumes.
+
+To use cross-database binding, you only need to use `*` to represent the database name when creating a binding. For example:
+
+```sql
+CREATE GLOBAL BINDING USING SELECT /*+ use_index(t, a) */ * FROM t; -- Create a GLOBAL scope standard binding.
+CREATE GLOBAL BINDING USING SELECT /*+ use_index(t, a) */ * FROM *.t; -- Create a GLOBAL scope cross-database binding.
+SHOW GLOBAL BINDINGS;
+```
+
+The output is as follows:
+
+```sql
++----------------------------+---------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+-----------------+--------+------------------------------------------------------------------+-------------+
```

Comment:
Maybe using `\G` instead of `;` to get vertical output would be more readable?

Reply:
Both are OK for me, but we always use `;` before, should we keep it compatible? @Oreoxmt

Reply:
To keep it consistent in this document, we can use `;`.